Sources/PackageDescription/PackageDescription.swift

```diff
@@ -426,9 +426,11 @@ public enum SystemPackageProvider {
   #if PACKAGE_DESCRIPTION_4
     case brewItem([String])
     case aptItem([String])
+    case yumItem([String])
```

Comment:
We shouldn't add this case for PackageDescription 4.

Reply:
ok